fieldapi_test.go

```diff
@@ -1,25 +1,125 @@
 package logr
 
-import "testing"
+import (
+	"errors"
+	"testing"
+	"time"
+)
+
+func TestFielInt(t *testing.T) {
```

Comment:
Minor typo in the method name. Should be "TestFieldInt".  Same for methods below.

Reply:
```suggestion
func TestFieldInt(t *testing.T) {
```